tapeagents/core.py

```diff
@@ -61,15 +61,15 @@ class PartialStep(BaseModel):
 
 
 class Observation(Step):
-    pass
+    kind: str = "observation"
 
 
 class Error(Observation):
     pass
 
 
 class AgentStep(Step):
-    pass
+    kind: str = "agent_step"
```

Comment:
can you make it of type Literal instead of str, just as all other steps, please.